src/hotspot/share/opto/addnode.hpp

```diff
@@ -51,7 +51,8 @@ class AddNode : public Node {
 
   // We also canonicalize the Node, moving constants to the right input,
   // and flatten expressions (so that 1+x+2 becomes x+3).
-  virtual Node *Ideal(PhaseGVN *phase, bool can_reshape);
+  virtual Node* Ideal(PhaseGVN *phase, bool can_reshape);
```

Comment:
Should be `PhaseGVN* phase` here?

Reply:
Thanks for looking at this. You're right. Fixed.